keyvalue/server/pom.xml

```diff
@@ -46,7 +46,12 @@
     </dependency>
     <dependency>
       <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-core-server</artifactId>
+      <artifactId>alluxio-core-server-master</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-server-worker</artifactId>
```

Comment:
so the convention would be that all direct dependencies of a module should be in its pom? I think we need to:

1) agree on the convention for which dependencies should go into pom
2) the order in which pom dependencies should be listed in the pom
3) a tool for enforcing the convention